ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPluginManager.m

```diff
@@ -35,7 +35,7 @@ - (CDVPlugin *)getCommandInstance:(NSString*)pluginName
 
   id obj = [self.pluginObjects objectForKey:className];
   if (!obj) {
-    obj = [[NSClassFromString(className)alloc] init];
+    obj = [[NSClassFromString(className)alloc] initWithWebViewEngine: self.webView];
```

Comment:
can you also do it on line 43?
https://github.com/ionic-team/capacitor/pull/4039/files#diff-c2302d00bf9f38fc9120a599208d0819d8baaccbe73a2e6c4aae3005e07eacadR43

Reply:
done